frontend/components/panels/DesktopSectionPanel.js

```diff
@@ -0,0 +1,142 @@
+/*
+ * This file is part of Search NEU and licensed under AGPL3.
+ * See the license file in the root folder for details.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import CSSModules from 'react-css-modules';
+import classNames from 'classnames/bind';
+
+import globe from './globe.svg';
+import css from './DesktopSectionPanel.css';
+import Keys from '../../../common/Keys';
+
+import LocationLinks from './LocationLinks';
+import WeekdayBoxes from './WeekdayBoxes';
+
+const cx = classNames.bind(css);
+
+
+class DesktopSectionPanel extends React.Component {
+  // Create the 4:35 - 5:40 pm string.
+  // This was copied from mobile section panel.js
+  // TODO: deduplicate
+  getTimeStingFromMeetings(meetingMoments) {
```

Comment:
Have we deduplicated this yet? Should we create BaseSectionPanel.js?

Reply:
Nope, and yeah!